include/flatbuffers/flatc_pch.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FLATBUFFERS_FLATC_PCH_H_
+#define FLATBUFFERS_FLATC_PCH_H_
+
+// code_generator.cpp
+// idl_parser.cpp
+#include <cmath>
+
+// code_generator.h
+//
+// cpp_generator.cpp
+// go_generator.cpp
+// idl_gen_go.cpp
+// swift_generator.cc
+// util.cpp
+#include <sstream>
+
+// idl_gen_dart.cpp
+// idl_gen_js_ts.cpp
+// python_generator.cc
+#include <cassert>
+
+// java_generator.h
+//
+// idl_gen_go.cpp
```

Comment:
While this is nice information to have, maintaining that his information is correct in the future will be tricky, and when it is not correct, it is not helpful, and we might as well not have it?